src/hb-set.h

```diff
@@ -60,126 +60,147 @@ typedef struct hb_set_t hb_set_t;
 HB_EXTERN hb_set_t *
 hb_set_create (void);
 
+/**
+ * Creates a set initially populated with num_codepoint codepoints, which must
+ * be in increasing order.
+ *
+ * Since: REPLACEME
+ */
+HB_EXTERN hb_set_t *
+hb_set_create_from_sorted (const hb_codepoint_t  *sorted_codepoints,
```

Comment:
Change this to hb_set_add_sorted_array(...) to mirror the underlying method. That has more utility because it can be used outside of set creation as well.